src/util/locale.js

```diff
@@ -121,5 +121,25 @@ export default {
      */
     formatDateTime: function(timestamp) {
         return moment(timestamp, 'X').format(this.getDateTimeFormat());
+    },
+
+    /**
+     * Determine direction for language
+     * @param {String} locale
+     * @return boolean
+     */
+    isLanguageRTL: function(locale) {
+        return [ ... this.getConfig().rtl || [] ].includes(locale);
```

Comment:
**nitpick:** While the global idea is good, it may have a small flaw in practice. Nothing forces the language name to be set in an exact case: it could be something like `en-us` or `en-US`. For this reason, it would be better to have a more permissive comparison. But yes, this prevents to use `includes`.

Anyway, most probably you cannot use this API as [it not supported by IE 11](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes), and I don't think we have a polyfill for it

**suggestion:** convert strings in lowercase and compare them to find correspondance